app/src/pages/inside/logsPage/logsPage.jsx

```diff
@@ -170,13 +170,13 @@ export class LogsPage extends Component {
   onHighlightRow = (highlightedRowId) => {
```

Comment:
Please, remove the code associated with the "highlight" from the logs page and related files.
Now it isn't used there.